src/components/planttracker/PlantTrackerSecondary.java

```diff
@@ -0,0 +1,5 @@
+package components.planttracker;
+
+public class PlantTrackerSecondary {
+    
```

Comment:
Do you not have any secondary methods? You could have some silly methods like `displayAllPlants()` or `getRandomPlant()` or even `listPlantsThatNeedWater()`.